### DISCUSSION TITLE
Is the PDF Association keeping an eye on the efforts to standardize post-quantum cryptography?

### DISCUSSION THREAD

**Discussion by fredericoschardong** [MatthiasValvekens/pyHanko#197] [General]
Hi @MatthiasValvekens,

I'm not sure if this is the proper place to start a discussion like this. If it isn't, please let me know so that we can move operations.

I haven't seen the sparkling new ISO 32001 and ISO 32002, but it appears that a tendency to stay up with the latest crypto primitives is in full swing. Is there a push to incorporate PQC signature algorithms into future PDF standards? What about hybrid signature methods that combine regular and PQC signatures? The latter, I believe, is less explored, although there is an expanding body of researchers advocating for the compatibility that hybrid systems bring to the table, which is critical for PDF.

**Comment by MatthiasValvekens** [comment]
Hi @fredericoschardong,


Good question! There are definitely no ongoing PDF-specific PQC projects right now, but in my experience people are generally aware of the developments in that space. I've had some conversations in the past about using hybrid signatures in timestamping applications, for example.

Having said that, it just doesn't make much sense to start working on an integration standard without a major standards body like NIST actually standardising the relevant cryptographic primitives... If NIST publishes standards for FALCON/Dilithium/SPHINCS+/... tomorrow, I'm sure it wouldn't take long to set the wheels in motion. Conversely, I don't think the process will start before that happens.

But it's an interesting question, and I've passed it on to the digital signatures technical working group in the PDF Association to solicit further opinions. :)
